### PR TITLE
[vsphere] use resource pool on 4.10 and later IPI installs

### PR DESCRIPTION
**Pull request by rvanderp3**
The intent of this PR is to install in to an existing resource pool for 4.10 and later IPI installs.

### CODE REVIEW

File: ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh [hunk 1]
```diff
@@ -57,6 +67,7 @@ platform:
     username: "${GOVC_USERNAME}"
     apiVIP: "${vips[0]}"
     ingressVIP: "${vips[1]}"
+    ${RESOURCE_POOL_DEF}
```
Comment (bostrt) [inline ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh:70]: Need to be `resourcePool: ${RESOURCE_POOL_DEF}`

File: ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh [hunk 2]
```diff
@@ -27,6 +27,16 @@ CONFIG="${SHARED_DIR}/install-config.yaml"
 base_domain=$(<"${SHARED_DIR}"/basedomain.txt)
 machine_cidr=$(<"${SHARED_DIR}"/machinecidr.txt)
 
+RESOURCE_POOL_DEF=""
+VERSION=$(echo "${JOB_NAME}" | grep -o -E '4\.[0-9]+')
+if [ ! -z ${VERSION} ]; then
+    Z_VERSION=$(echo ${VERSION} | cut -d'.' -f2)
+    if [ ${Z_VERSION} -gt 9 ]; then
+        echo "4.x installation is later than 4.9, will install with resource pool"
+        RESOURCE_POOL_DEF="ipi-ci-clusters"
```
Comment (bostrt) [inline ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh:36]: I think this needs to be `RESOURCE_POOL_DEF="resourcePool: ipi-ci-clusters"`

Comment (rvanderp3) [reply ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh:36]: yep, good catch!

**Comment by rvanderp3** [comment]
confirmed that ipi installs > 4.9 are landing in the `ipi-ci-clusters` resource pool in both VMC and IBM Cloud.  failing tests appear to be unrelated to this change.

**Pull request merged**